src/helpers/i18n.js

```diff
@@ -27,9 +27,11 @@ export function formatAmount(intl, amount) {
 }
 
 export function formatDateFromISO(mm, intl, date) {
-  let isSecondaryCalendar = JSON.parse(localStorage.getItem('isSecondaryCalendarEnabled'));
+  const isSecondaryCalendar = JSON.parse(localStorage.getItem("isSecondaryCalendarEnabled"));
   if (isSecondaryCalendar) {
-    return neFormateDateFromISO(date);
+    const secondCalendarFormatting = mm.getConf("fe-core", "secondCalendarFormatting", "DD-MM-YYYY");
```

Comment:
"DD-MM-YYYY" could be moved to consts:
`
const DEFAULT_CALENDAR_FORMAT = "DD-MM-YYYY";
`

Reply:
In my opinion it makes it less readable and complicated so I did not apply this changes. Additionally if we move those settings to a separate file (as planned) it would be redundant. Those values are only fallbacks for settings fetched from db.
@delcroip @dragos-dobre 
What do you think?

Reply:
Utilizing const enhances readability by clearly indicating that "DD-MM-YYYY" is the default value, which is particularly helpful for those unfamiliar with the system. Additionally, using const improves reliability by minimizing the risk of typographical errors in the string value. @sniedzielski was suggesting changes like this in my code, so I do it too.

Reply:
I totally agree with @olewandowski1 . Please @wzglinieckisoldevelo use constants within application where it is possible, Thanks to constansts it is easier to maintains code in case of any potential issues.

Reply:
I disagree in this aspect it also makes codes more readable. You can clearly see the default value, if you move it to constant you have to check two files to know that. Let's compare two versions:
`   const varA = mm.getConf("fe-core", "varA", true);
    const varB = mm.getConf("fe-core", "varB", DEFAULT_VAR_B);`
    
I believe version A is much more readable. If you are starting your work with the system you will learn that pretty quickly, and it is used in many different technologies (fallback value) so not something unusual. And about newcomers - we define redux state using all caps letters, so this could be more confusing for people that have no experience with openIMIS or frontend in general.

If we leave some constant value in the code and then use something different from the db it would be more confusing. If you spot DEFAULT_VAR_B in constant.js and then see in the app that actually used value is different it would be harder to understand, not easier.

Reply:
I not sure I will help here

it is strange for me , there is a mix of neFormateDateFromISO and SecondFormateDateFromISO:

I believe the specificity of the neCalandar should be set in neDateFormater.

Then when the function is called  in  'ne' FormateDateFromISO , if the fallback date formatting can be overruled by config then I don't see the issue of hardcoding it (I checked getConf and most of the fallback value are hardcoded)

in i18n a simple SecondFormateDateFromISO() with no argurment could be called after the if(secondCalandar) as formating details could be set in  NeDateFormatter.js